packages/vue/src/components/molecules/SfPagination/SfPagination.vue

```diff
@@ -47,16 +50,17 @@
         </li>
         <li class="sf-pagination__item">
           <slot name="number" v-bind="{ go, number: total }">
-            <button class="sf-pagination__button" @click="go(total)">
+            <SfLink class="sf-pagination__button" @click="go(total)">
```

Comment:
@justyna-13 for SfLink we should always use props `link` because it should be accessible without JS. But now I don't have idea how to handle it.